modules/data/redis-cache/main.bicep

```diff
@@ -0,0 +1,250 @@
+@description('Required. The name of the Redis cache resource.')
+param name string
```

Comment:
```suggestion
@description('The prefix of the Redis cache resource name.')
param prefix string = 'redis-'

@description('The name of the Redis cache resource.')
param name string = '${prefix}${uniqueString(resourceGroup().id, location)}'
```

Reply:
We can make the name optional. We standardize our prefixes per this [doc](https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-naming).

Here is the way I do it, breaking up the name into a prefix which makes it easy to override.

Reply:
Done